src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
```

Comment:
Capitalize the names of this constant, they should also be in camel case (same as it is in the checklist).

By the way, such a constants can be outside of the function ;)